src/Modules/Shared/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psm1

```diff
@@ -1,174 +0,0 @@
-

```

Comment:
this file name is referenced 3 times:
* /src/Modules/map.json (remove entry?)
* /src/System.Management.Automation/engine/InitialSessionState.cs (replace with some other build result file?)
* /src/System.Management.Automation/security/Authenticode.cs (replace with some other build result file?)